OutOfSchool/OutOfSchool.WebApi.Tests/Extensions/CATOTTGAddressExtensionsTests.cs

```diff
@@ -220,6 +233,19 @@ public void GetDistrictName_WhenCATOTTGIsCityDistrictWithParent_ShouldReturnDist
         Assert.AreEqual(expected, result);
     }
 
+    [Test]
+    public void GetDistrictName_WhenCATOTTGIsCityDistrictWithSpecialStatusParent_ShouldReturnNull()
+    {
+        // Arrange
+        string expected = null;
+
+        // Act
+        var result = CatottgAddressExtensions.GetDistrictName(cityDistrictWithSpecialStatusParent);
+
+        // Assert
+        Assert.AreEqual(expected, result);
```

Comment:
(Code style) `Assert.IsNull` is more clear, and you could remove 3 lines of code of you `Arrange` part. Less code, more clarity

Reply:
I've changed them. Thank you!